filebeat/harvester/log.go

```diff
@@ -258,16 +248,20 @@ func (h *Harvester) close() {
 	// On completion, push offset so we can continue where we left off if we relaunch on the same file
 	h.sendStateUpdate()
 
-	logp.Debug("harvester", "Stopping harvester for file: %s", h.path)
+	logp.Debug("harvester", "Stopping harvester for file: %s", h.state.Source)
 
 	// Make sure file is closed as soon as harvester exits
 	// If file was never opened, it can't be closed
 	if h.file != nil {
 		h.file.Close()
-		logp.Debug("harvester", "Stopping harvester, closing file: %s", h.path)
+		logp.Debug("harvester", "Stopping harvester, closing file: %s", h.state.Source)
+		harvesterOpenFiles.Add(-1)
```

Comment:
there is a check `h.file != nil` right before `h.file.Close()`. So `h.file == nil` only if file has not been opened before? Can harvester be closed twice?